dcos/marathon.py

```diff
@@ -22,10 +22,9 @@ def create_client(config=None):
         config = util.get_config()
 
     marathon_url = _get_marathon_url(config)
-    timeout = config.get('core.timeout', http.DEFAULT_TIMEOUT)
 
     logger.info('Creating marathon client with: %r', marathon_url)
-    return Client(marathon_url, timeout)
+    return Client(marathon_url)
```

Comment:
I think we were trying to avoid reading and parsing the config file many times. cc @tamarrow.

Also, ideally I would like us to make the modules in the python package `dcos` able to work outside of the CLI. Importing `dcos.config` in module in the package `dcos` will make this difficult.